docs/migration.md

```diff
@@ -353,37 +396,11 @@ service:
 
 #### Name
 
-Define the name after the slash `/` in the receiver name.
```

Comment:
I guess we must do something similar for all the other sources? Could we also move `Source Category` and `Fields` to a common section?

Reply:
Sure, i just won't be able to deliver it Today :sweat_smile:

Reply:
Wanted to show the direction we have been talking about Yesterday